pkg/kubefedctl/federate/util.go

```diff
@@ -126,7 +131,19 @@ func namespacedAPIResourceMap(config *rest.Config, skipAPIResourceNames string)
 	return apiResources, nil
 }
 
-func apiResourceMatchesSkipNames(apiResource metav1.APIResource, skipAPIResourceNames, group string) bool {
+func apiResourceGroupMatchesSkipName(skipAPIResourceNames, group string) bool {
+	for _, name := range strings.Split(skipAPIResourceNames, ",") {
```

Comment:
As per my comment on #741, unmarshalling (and validation) of user input should generally be done in `Complete`.  This won't matter, though, if you use `StringSliceP` instead of `StringVarP`.

Reply:
Updated to `StringSliceVarP`.